cmd/env.go

```diff
@@ -95,18 +105,19 @@ func envRun(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	fmt.Printf("export AWS_ACCESS_KEY_ID=%s\n", shellescape.Quote(creds.AccessKeyID))
-	fmt.Printf("export AWS_SECRET_ACCESS_KEY=%s\n", shellescape.Quote(creds.SecretAccessKey))
-	fmt.Printf("export AWS_OKTA_PROFILE=%s\n", shellescape.Quote(profile))
+	printExport("AWS_ACCESS_KEY_ID", shellescape.Quote(creds.AccessKeyID))
```

Comment:
Move the Quote into the function to reduce repetition.

Reply:
Actually, I don't know fish that well. Will the `shellescape` package still work?